environments/nanopore.yml

```diff
@@ -6,4 +6,7 @@ dependencies:
   - samtools
   - minimap2
   - ont-fast5-api
-  - csvtk
\ No newline at end of file
+  - csvtk
+  - pip
+  - pip:
+    - pysam
```

Comment:
Unless there are specific config-y reasons, it is probably a good idea to end this with a newline